ramlfications/loader.py

```diff
@@ -28,7 +28,10 @@ def _yaml_include(self, loader, node):
         file_name = os.path.join(os.path.dirname(loader.name), node.value)
 
         with open(file_name) as inputfile:
-            return yaml.load(inputfile)
+            if file_name.endswith(('yaml', 'raml', 'json')):
+                return yaml.load(inputfile, self._ordered_loader)
```

Comment:
Here I would actually return `self._ordered_load(inputfile, yaml.SafeLoader)`, and getting rid of the added lines 49-51